src/pack.rs

```diff
@@ -209,6 +277,11 @@ pub async fn pack(options: PackOptions) -> Result<()> {
             .await
             .map_err(|e| anyhow!("could not copy file to channel directory: {}", e))?;
 
+        if let Some(parent) = path.parent()
```

Comment:
I think this should always be `Some` so we can also use `path.parent().expect("Parent exists")`